docs/prebuilt_models.rst

```diff
@@ -466,7 +466,7 @@ Mistral
     - 
```

Comment:
Could we make corresponding changes and uncomment 
https://github.com/mlc-ai/mlc-llm/blob/main/docs/prebuilt_models.rst?plain=1#L98-L210

The idea is to give users some quick steps to follow for using prebuilt models on iOS and point to the main iOS page, something similar to Android. We can refer to the previous prebuilt_model page: https://llm.mlc.ai/docs/prebuilt_models_deprecated.html#prebuilt-models-on-ios.